docs/install/install_instructions.md

```diff
@@ -199,7 +199,9 @@ In order to build NEURON from source, the following packages must be available:
 
 - Bison
 - Flex >= 2.6
-- C/C++ compiler suite supporting C++17
+- C/C++ compiler suite supporting C++17 (e.g. GCC >=9.3.1, Clang >= 11.0.0)
+  - Note that some C++17 features require a newer compiler version.
+  - Features that require extra linking are not supported, such as `std::filesystem`, where we'd need to link against the `stdc++fs` library.
```

Comment:
I don't understand what you mean there.
We do not support <filesystem>?

Reply:
we support it, but not through extra libs (stdc++fs , which is remnant of std::experimental::filesystem)